test/eth/GasEstimatorService.spec.js

```diff
@@ -17,142 +22,52 @@ function buildTestGasEstimatorService() {
   return service;
 }
 
-test('policies initially null', done => {
-  const gasEstimator = buildTestGasEstimatorService();
-
-  gasEstimator
-    .manager()
-    .connect()
-    .then(() => {
-      expect(gasEstimator.getPercentage()).toBe(null);
-      expect(gasEstimator.getAbsolute()).toBe(null);
-      done();
-    });
+test('policies initially null', () => {
```

Comment:
super minor suggestion: might as well check the fallback value too.  Can also change the name of the test